appservice/src/main/java/nu/yona/server/messaging/rest/MessageController.java

```diff
@@ -340,6 +342,15 @@ private void addSenderBuddyLinkIfAvailable(MessageDto message)
 			}
 		}
 
+		private void addSenderUserPhotoLinkIfAvailable(MessageDto message)
+		{
+			if (message.getSenderUserPhotoId().isPresent())
```

Comment:
This can be done simpler. I've refactored ``addSenderBuddyLinkIfAvailable`` as an example:

```
		private void addSenderBuddyLinkIfAvailable(MessageDto message)
		{
			message.getSenderBuddyId().ifPresent(buddyId -> message.add(
					BuddyController.getBuddyLinkBuilder(goalIdMapping.getUserId(), buddyId).withRel(BuddyController.BUDDY_LINK)));
		}
```

Reply:
Will simplify